build_tools/travis/test_script.sh

```diff
@@ -23,6 +23,10 @@ run_tests() {
     EXIT_STATUS="$(($EXIT_STATUS+STATUS))"
   done
 
+  python -m pytest -v -k "test_transforms"
```

Comment:
I meant run every test with pytest. the for loop above iterates over every test with unittest currently, just modify it to pytest

Reply:
ooh gotcha, thought you meant just for the tests we introduced. I'll change it.